### ISSUE TITLE
emitting without correct spacing

### ISSUE THREAD

**Issue by dariajung** [angular/ts-minify#34]
instead of emitting as `private X()`, private methods are being emitted like so: `privateX`

**Comment by dariajung** [comment]
Could be as simple as adding

```
case ts.SyntaxKind.StaticKeyword: 
      case ts.SyntaxKind.PrivateKeyword: {
        return (node.getText() + ' ');
      }
```

but there are lots of keywords.

**Comment by dariajung** [comment]
Fixed with 370532b52acc766addc1eb09b315cc21ca138669
